frontend/src/layout/lemonade/TopBar/SitePopover.tsx

```diff
@@ -5,7 +5,14 @@ import { userLogic } from '../../../scenes/userLogic'
 import { ProfilePicture } from '../../../lib/components/ProfilePicture'
 import { LemonButton } from '../../../lib/components/LemonButton'
 import { LemonRow } from '../../../lib/components/LemonRow'
-import { IconCheckmark, IconOffline, IconPlus, IconSignOut, IconUpdate } from '../../../lib/components/icons'
+import {
+    IconCheckmark,
+    IconOffline,
+    IconPlus,
+    IconSignOut,
+    IconUpdate,
+    IconExclamation,
+} from '../../../lib/components/icons'
```

Comment:
Nit: Prefer absolute paths

```suggestion
} from 'lib/components/icons'
```